First Contact tweaks.

Flaksim opened the pull request:
*Planetary map significantly enlarged, some positions moved around to encourage more varied gameplay.
*Sterile gloves added to the corvette and other surgery rooms.
*URF gets some love! BR85's and ammunition added to their vessel.
*Civilian ship crew spawn fixed.

Flaksim committed:
First Contact tweaks.

*Planetary map significantly enlarged, some positions moved around to encourage more varied gameplay.
*Sterile gloves added to the corvette and other surgery rooms.
*URF gets some love! BR85's and ammunition added to their vessel.
*Civilian ship crew spawn fixed.

Flaksim committed:
Corvette fix.

*Add paperbin for use with fax.
*Added pen for use with fax.

Flaksim committed:
First Contact Tweaks

*Adds trashcans to the research base.
*Adds several supplies to the supply room.
*Adds several "manuals" on "science".

Flaksim committed:
Corvette fix.

*Adds a terminal to the corvette which "should" fix MAC issues.
*Fixed phoron tank dir's, half of them werent feeding into the pipes.

Flaksim committed:
Fixes to CCV Comet and Exoresearch 1

*Added toolboxes and the like to exoresearch 1.
*Replaced BR85 with MA37 on CCV Comet.

Flaksim committed:
Job Tweaks

*ONI Officer is now an ODST Squad Leader.
*Equipment and rank changed accordingly.
*Further equipment changes to exoresearch.

Flaksim committed:
Corvette fuel tweak.

Increases corvette fuel

Flaksim committed:
Exoplanet Medical tweak.

*Adds Surgery supplies to exoplanet medical area.

Flaksim committed:
Labeling update.

Down to the last detail tweaks it seems:
*Labeled all closets and crates to denote contents.
*Added paper shredders, clipboards, bookcase, some more random supplies for science.

Flaksim committed:
Travis fix.

Attempt to fix travis bug, fuel supply reduced to pre patch level.

Flaksim committed:
Disable ship weapons.

This commit removes the MAC weapon controls from the UNSC and URF ships.
(The controls are not gone, merely moved out of reach of the players)

This is done so first contact can be properly tested giving the covenant a chance, pending the addition of defensive measures for their ship and different weapon systems for small covenant and UNSC/URF vessels.

Flaksim committed:
Travis fix

Travis acted up when I moved the consoles, no MAC guns work either way, put em back.

Flaksim committed:
Vendor fix.

*Fixes marine vendor
*Moves communications hub around on the map, changes to the cave system to accomodate.

Flaksim committed:
Adds new compound and role.

*Remote communications center added to research site.
*Staffed by two Communications Operators. E3 ranked marines whom operate and guard the compound
*Added colors for the factions in the role setup menu.
*Removed marine vendor on research, added shotgun ammunition to ODST vendor.

Flaksim committed:
Color change

Makes covenant job color more readable.

Flaksim committed:
Covenant map changes

*Altered the covenant ship interior layout significantly.

Flaksim committed:
Last commit this PR is DONE!

*Adds an observation pen to exoresearch
*New spawnpoint for Unggoy
*Adds a holding cell to Covenant ship
*New Unggnoy role: Junior crewmember, minor gear.
*Restraints added to certain location to facilitate capture under certain conditions.

Flaksim committed:
Forgot a button, now its DONE

Added shutter button.

bloxgate commented on maps/covenant_ships/kig_yar_pirates/spawns_jobs.dm:
```diff
@@ -90,24 +107,51 @@ GLOBAL_LIST_EMPTY(kigyar_pirate_spawns)
+	spawn_positions = 3
+	selection_color = "#800080"
+	outfit_type = /decl/hierarchy/outfit/unggoy
+	access = list(230,250)
```
Why are these the actual numbers? Shouldn't there be defines for these?

Flaksim replied:
I asked XO about that, He told me it'd also work that way.
It does work that way aswell, you're welcome to make a file with defines for this gamemode for each job, and when that is in and I will modify the maps accordingly.

But I wasnt going to sit around for a coder to do something.

In practice however, its superfluous work, making a file containing the defines and referencing those instead of what I did with the numbers is exactly the same thing.

Flaksim committed:
Last minute map addition!

*New planet added! An Icy world containing a remote UNSC Fuel/Supplies depot.
*Staffed by three depot guards (Lance corporals)
*Two Z levels, has an internal hangar bay for corvette pelican docking.
*Changed planet names to random designations, science outpost is now "VT9-042", Ice depot is KS7-535.

Tested, map works and spawns on the overmap.

Suggestion: Given the number of ships and now a second planet, the overmap size should be expanded, I do not know how to mess with global vars properly, so will leave that to a coder.

Flaksim committed:
Clothing additions.

*Added winter jackets and flashlights for going outside the facility on ice.

Flaksim committed:
Depot guard spawn fix

*Mislabeled something, fixed.
*Altered corvette icon, its now more sleek and compact like UNSC corvettes tend to be.

Flaksim committed:
Further FC tweaks.

*Biofoam Syringe renamed to Biofoam Canister, Icon altered aswell to be more suitable.
*Medical Supplies added to the supply depot.
*Overmap size tripled in the hopes of reducing ships and planets spawning extremely close to each other.
*Medical supplies added to Exoresearch.
*Some animals added to Exoresearch exterior.

Flaksim committed:
prep for deck guns.

Prep for deck guns+ added binoculars to some locations.

Flaksim committed:
overmap area holder fix

fixes overmap area holder

Flaksim committed:
area fix

area fix

X0-11 committed:
Merge branch 'alpha' into alpha

Flaksim committed:
Deck guns on human armed vessels

*Corvette gets 8 regular guns, innie ship 1 chaingun.
